classes/RequestSql.php

```diff
@@ -239,8 +239,20 @@ public function cutJoin($attrs, $from)
             if (in_array($attr['expr_type'], ['operator', 'const'])) {
                 continue;
             }
-            if ($attribut = $this->cutAttribute($attr['base_expr'], $from)) {
-                $tab[] = $attribut;
+
+            if (!empty($attr['sub_tree'])) {
+                foreach ($attr['sub_tree'] as $treeItem) {
+                    if ($treeItem['expr_type'] !== 'colref') {
```

Comment:
If I understand well, colref is a reference for DB column. Can you make a test with the request in the issue ( sthg like a.x = b.x AND 1) ?

Reply:
I have added a case ;)